src/components/StabilityFeeAlert.jsx

```diff
@@ -0,0 +1,34 @@
+// Libraries
+import React from "react";
+import {inject, observer} from "mobx-react";
+
+// Components
+import InlineNotification from "./InlineNotification";
+
+@inject("content")
+@inject("network")
+@observer
+class StabilityFeeAlert extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = { show: true }
+  }
+
+  render() {
+    return (
+      this.props.content.showStabilityFeeAlert() &&
+        this.state.show &&
+        !localStorage.getItem(`StabilityFeeChangeAlertClosed-${this.props.content.stabilityFeeMarkdown()}`) &&
```

Comment:
these 3 checks should really just be `this.props.content.showStabilityFeeAlert()`

Reply:
That can be changed but I was following the pattern from https://github.com/makerdao/scd-cdp-portal/blob/develop/src/components/LegacyCupsAlert.jsx